recipe/meta.yaml

```diff
@@ -12,16 +12,18 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
-    - python >=3.4
+    - python
     - pip
+    - wheel
+    - setuptools
     - setuptools_scm
+    - tomli
```

Comment:
```suggestion
    - toml
```
The correct will be `toml` not `tomli`. It's needed only for backward compatibility https://github.com/pypa/setuptools_scm/blob/e9cbb5a68b3ae6d5c549bda293ef60bb5ec8ec7e/pyproject.toml#L61-L62 or it can be dropped.
Python 3.11 already has `tomlib` library by defaults

Reply:
ah but this is `amply` not `setuptools_scm`